app/models/host/managed.rb

```diff
@@ -462,6 +462,16 @@ def host_params
     @cached_host_params = hp
   end
 
+  def host_inherited_params_objects
```

Comment:
Code above returns hash of the inherited params
This method returns the objects.
I we don't care about the objects, I can use the hashes from the code above.